cmd/bbolt/main.go

```diff
@@ -1796,23 +1796,32 @@ func ReadPage(path string, pageID int) (*page, []byte, error) {
 
 	// Determine total number of blocks.
 	p := (*page)(unsafe.Pointer(&buf[0]))
+	if p.id != pgid(pageID) {
+		return nil, nil, fmt.Errorf("error: %w due to unexpected page id: %d != %d", ErrCorrupt, p.id, pageID)
+	}
 	overflowN := p.overflow
+	if overflowN > 100*1024 {
```

Comment:
It means bbolt doesn't support saving object > 400MB (assuming pageSize is 4K)?

Reply:
Addresses in the other PR: https://github.com/etcd-io/bbolt/pull/354